src/Nri/Ui/Pennant/V3.elm

```diff
@@ -0,0 +1,161 @@
+module Nri.Ui.Pennant.V3 exposing (contentPremiumFlag, activePremiumFlag, inactivePremiumFlag, expiredPremiumFlag, giftPremiumFlag)
+
+{-| Changes from V3:
```

Comment:
```suggestion
{-| Changes from V2:
```

Reply:
Regarding the `contentPremiumFlag`, the icon is supposed to be smaller by default. Is that a difference we can keep if I update the docs for the component to mention this explicitly?